packages/instrumentation-aws-sdk/src/utils.ts

```diff
@@ -14,9 +14,10 @@ export const removeSuffixFromStringIfExists = (str: string, suffixToRemove: stri
 
 export const normalizeV2Request = (awsV2Request: Request<any, any>): NormalizedRequest => {
     const service = (awsV2Request as any)?.service;
+    const operation = (awsV2Request as any).operation;
```

Comment:
In the above line we use the `?.` optional chaining operator to protect against `awsV2Request` being nullish.
I can see that the original code was not using it, but it probably better to have it and stay consistent

Reply:
Sounds good! I will add the operator!